client/js/components/procedure/StatementSegmentsList/StatementMeta/StatementMetaLocationAndDocumentReference.vue

```diff
@@ -0,0 +1,365 @@
+<template>
+  <fieldset data-dp-validate="statementLocationAndDocumentReference">
+    <legend
+      id="locationAndDocuments"
+      class="mb-3 color-text-muted font-normal">
+      {{ Translator.trans( hasPermission('field_statement_polygon') ? 'location.and.document.reference' : 'document.reference') }}
+    </legend>
+
+    <!-- Location reference -->
+    <div
+      v-if="hasPermission('field_statement_polygon')"
+      class="font-semibold mb-1">
+      {{ Translator.trans('location') }}
+    </div>
+
+    <div class="space-y-4">
+      <div v-if="hasPermission('field_statement_polygon')">
```

Comment:
nitpick: Do we need to render the div (for e.g. styling purposes or can this be a `<template>`-tag, too? With `<template>` there are less DOM nodes that needs to be rendered in the Browser.

Reply:
You're right, we don't need the div. Replaced in d331046945dbc4e2a6a7db7f5b2ad69823d9de44